packages/application-components/src/components/modal-pages/internals/modal-page.tsx

```diff
@@ -61,6 +61,10 @@ type MessageDescriptor = {
   defaultMessage?: string;
 };
 type Label = string | MessageDescriptor;
+
+// naming of `xlarge` should be discussed, I didn't know what to name it
+export type TModalPageSize = 'small' | 'large' | 'xlarge';
```

Comment:
We are moving towards always using numeric range scales for size props.
Can we maybe use a different set of values here?
```suggestion
export type TModalPageSize = 10 | 20 | 30;
```